### PR TITLE
Fill projects liveness configs

### PR DESCRIPTION
**Pull request by maciekzygmunt**
Resolves L2B-2959

### CODE REVIEW

**Review by antooni — CHANGES_REQUESTED** [review]
Check sinceTimestamp:
- transfers - get Timestamp of the first transfer from to
- functionCalls :
 - try to get first call of this method (Etherscan internal calls API?)
 - if you cannot, then check all the timestamps added whether they are all reasonably old

File: packages/config/src/layer2s/metis.ts [hunk 1]
```diff
@@ -65,6 +65,31 @@ export const metis: Layer2 = {
       url: 'https://andromeda.metis.io/',
       startBlock: 1,
     },
+    liveness: {
+      batchSubmissions: [
+        {
```
Comment (antooni) [inline packages/config/src/layer2s/metis.ts:70]: delete it

File: packages/config/src/layer2s/nova.ts [hunk 2]
```diff
@@ -103,6 +103,32 @@ export const nova: Layer2 = {
       assessCount: subtractOne,
       startBlock: 1,
     },
+    liveness: {
+      batchSubmissions: [
+        {
+          formula: 'functionCall',
```
Comment (antooni) [inline packages/config/src/layer2s/nova.ts:109]: should we add 2 more methods to batchSubmissions

File: packages/config/src/layer2s/linea.ts [hunk 3]
```diff
@@ -116,6 +116,21 @@ export const linea: Layer2 = {
       type: 'rpc',
       startBlock: 1,
     },
+    liveness: {
+      batchSubmissions: [],
```
Comment (antooni) [inline packages/config/src/layer2s/linea.ts:120]: is it posting txs?

File: packages/config/src/layer2s/polygonzkevm.ts [hunk 4]
```diff
@@ -104,6 +104,32 @@ export const polygonzkevm: Layer2 = {
       type: 'rpc',
       startBlock: 1,
     },
+    liveness: {
```
Comment (antooni) [inline packages/config/src/layer2s/polygonzkevm.ts:107]: double check

File: packages/config/src/layer2s/arbitrum.ts [hunk 5]
```diff
@@ -327,6 +327,32 @@ export const arbitrum: Layer2 = {
       assessCount: subtractOneAfterBlockInclusive(22207818),
       startBlock: 1,
     },
+    liveness: {
+      batchSubmissions: [
```
Comment (antooni) [inline packages/config/src/layer2s/arbitrum.ts:331]: should we add 2 more?

**Review by torztomasz — APPROVED** [review]

**Pull request merged**